dotnet-engine/Yggdrasil.Engine.Tests/YggdrasilEngineTest.cs

```diff
@@ -100,6 +102,54 @@ public void TestClientSpec()
         }
     }
 
+    [Test]
+    public void Custom_Strategies_Required_But_Not_Configured_Fails() {

```

Comment:
I don't think this is a fail, this should degrade gracefully

```suggestion
    public void Custom_Strategies_Required_But_Not_Configured_Returns_False() {

```